app/build.gradle

```diff
@@ -71,7 +71,7 @@ dependencies {
     implementation 'com.google.android.gms:play-services-location:18.0.0'
     implementation "androidx.fragment:fragment:1.3.2"
     implementation 'androidx.room:room-runtime:2.2.6'
-    implementation 'com.google.firebase:firebase-database:18.0.1'
+    implementation 'com.google.firebase:firebase-database:19.5.1'
```

Comment:
Changing the version worked for me, so I'm really surprised you still have the Firebase SDK version error from Cirrus... I also tried using the latest version (i.e 19.7.0), but it still wasn't working.

Reply:
Yes, and my first commit works, so I don't understand why it's fail now. Last week  with Jean-Luc and Anthony, we already try to comment the test, but an other test in the chat produce this error.